pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm

```diff
@@ -128,41 +142,15 @@ sub _process_template {
     die "Template processing failed: $output";
 }
 
-sub create_purchase_link {
```

Comment:
I am not sure why are we deleting this. By deleting this subroutine we don't have the custom URL functionality anymore, also we are deleting the message when the user has Imunify installed but not licensed.

Reply:
We could do something similar to my comment in line 155.
Just updating the variables so we don't refactor too much code.
```
    my $installed = Whostmgr::Imunify360::is_imunify360_installed()
    my $price = Whostmgr::Imunify360::get_imunify360_price();
```

Reply:
This was deleted as part of the revert, so it looks like this was a v82-only change. This won't be an immediate problem for v80, but you're right that it will need to be restored for some of our v82 functionality to be complete.